skimage/data/__init__.py

```diff
@@ -9,6 +9,7 @@
 import os as _os
 
 import numpy as _np
+from warnings import warn
 
 from .._shared._warnings import expected_warnings, warn
```

Comment:
lets remove this line? when did it sneak in....

Reply:
ohoh!